assets/pages/digital-subscription-landing/components/priceCtaContainer.jsx

```diff
@@ -9,21 +9,22 @@ import PriceCta from 'components/priceCta/priceCta';
 import { digitalSubPrices } from 'helpers/subscriptions';
 import { getDigitalCheckout } from 'helpers/externalLinks';
 import type { CommonState } from 'helpers/page/page';
+import { currencies } from 'helpers/internationalisation/currency';
 
 
 // ----- State Maps ----- //
 
 function mapStateToProps(state: { common: CommonState }) {
-
-  const price = digitalSubPrices[state.common.countryGroup].toFixed(2);
+  const { countryGroupId } = state.common.internationalisation;
+  const price = digitalSubPrices[countryGroupId].toFixed(2);
 
   return {
     ctaText: 'Start a 14 day free trial',
     url: getDigitalCheckout(
       state.common.referrerAcquisitionData,
-      state.common.countryGroup,
+      state.common.internationalisation.countryGroupId,
```

Comment:
You've got this from line 18 haven't you?

Reply:
Fixed here: c032fbc